R/commonDiscoverDistributions.R

```diff
@@ -962,6 +962,7 @@
   }
 
   lineData <- ggplot2::layer_data(plot, 1)
+  lineData <- lineData[!is.infinite(lineData$y),]
```

Comment:
if you also want to exclude NA and NaN you could use `is.finite`.
``` r
u <- c(NA, NaN, -Inf, Inf)
names(u) <- u
!is.infinite(u)
#>  <NA>   NaN  -Inf   Inf 
#>  TRUE  TRUE FALSE FALSE
is.finite(u)
#>  <NA>   NaN  -Inf   Inf 
#> FALSE FALSE FALSE FALSE
```

Reply:
Thanks!